Content.Server/Throw/ThrowHelper.cs

```diff
@@ -83,7 +83,9 @@ public static void Throw(IEntity thrownEnt, float throwForce, GridCoordinates ta
                 physComp = thrownEnt.AddComponent<PhysicsComponent>();
 
             var timing = IoCManager.Resolve<IGameTiming>();
-            var spd = throwForce / (1f / timing.TickRate); // acceleration is applied in 1 tick instead of 1 second, scale appropriately
+
+            // scaling is handled elsewhere, this is just multiplying by 60 independent of timing as a fix until elsewhere values are updated
+            var spd = throwForce * 60;
```

Comment:
We should find the existing sources of throw force, multiply them all by 60, then get rid of this in a later PR.